tools/workspace-plugin/src/executors/build/lib/babel.ts

```diff
@@ -50,7 +50,12 @@ export async function compileWithGriffelStylesAOT(options: NormalizedOptions) {
     return processAsyncQueue(compilationQueue);
   }
 
-  await compileSwc(esmConfig, options);
+  const transforms = [];
+  if (options.enableGriffelRawStyles) {
+    logger.log('💅 Griffel RAW styles output enabled');
+    transforms.push(createStyleRawOutput);
+  }
+  await compileSwc(esmConfig, options, transforms);
```

Comment:
this is now explicit - this transform occurs only if babel griffel processing is on and enableGriffelRawStyles is enabled in user config